### ISSUE TITLE
Multisite with binary

### ISSUE THREAD

**Issue by te-deum** [dunglas/frankenphp#862]
### Describe you feature request

I would like to know if it's possible to host multiple website with the FrankenPHP binary like Caddy can do with a JSON configuration file ?

My project use Laravel and is a SPA with Vue. I use a load balancer as reverse proxy to manage TLS connexion to my project. I actually use Nginx as HTML server and everything is working well.

I want to try and test FrankenPHP but I can't get multiple site working with the binary. I try to give a config JSON file without success :-(
Here is the simple JSON that I try :
```
:8081 {
    root * /data/app/public
    encode zstd gzip br
}

:8082 {
    root * /data/demo/public
    encode zstd gzip br
}

:8083 {
    root * /data/beta/public
    encode zstd gzip br
}
```

Any idea if it's possible to do what I want ?

**Comment by withinboredom** [comment]
If you are running them all on different ports, why not run a separate process for each of them?

**Comment by dunglas** [comment]
Yes. It's just a Caddy module, all Caddy features are supported.

**Comment by rikwillems** [comment]
@te-deum see the Caddyfile that FrankenPHP uses: [https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/Caddyfile](https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/Caddyfile)

You can use the environment variable `CADDY_EXTRA_CONFIG` to inject what you want. Or mount your own Caddyfile.

**Comment by henderkes** [comment]
```Caddyfile
:8081 {
    root * /data/app/public
    encode zstd gzip br
    php_server
}

:8082 {
    root * /data/demo/public
    encode zstd gzip br
    php_server
}

:8083 {
    root * /data/beta/public
    encode zstd gzip br
    php_server
}
```

Just like that.
